### PR TITLE
[Term Entry] PyTorch Tensor Operations: .mm()

### LINKED ISSUE #5059: [Term Entry] PyTorch Tensors .mm()
### Reason/inspiration (optional)

Add a new term entry about the `.mm()` method for Tensors. Place your file under the file path `docs/content/pytorch/concepts/tensor-operations/terms/mm/mm.md` 

It should include the following sections:

- A "Syntax" section with a \```pseudo code block
- An "Example" section with a \```py code block
- A "Codebyte Example" section with a \```codebyte/python code block

Please refer to the [concept entry templates](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Sriparno08**
### Description

Created a new term entry for the `.mm()` method under `Tensor Operations` in PyTorch.

### Issue Solved

Closes #5059

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/pytorch/concepts/tensor-operations/terms/mm/mm.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+```pseudo
+torch.mm(ten1, ten2, *, out=None)
+```
+
+- `ten1`: The first tensor to be multiplied.
+- `ten2`: The second tensor to be multiplied.
+- `out` (Optional): The output tensor to be used. The default value is `None`.
+
+If `ten1` is a `(m x n)` tensor and `ten2` is a `(n x p)` tensor, then `out` will be a `(m x p)` tensor.
```
Comment (cigar-galaxy82) [inline content/pytorch/concepts/tensor-operations/terms/mm/mm.md:29]: mentioned this as a note

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @Sriparno08 thank you for contributing to Codecademy docs, I have suggested a single changes please modify them.

**Comment by Sriparno08** [comment]
Done, @cigar-galaxy82.

**Review by cigar-galaxy82 — APPROVED** [review]

**Comment by mamtawardhani** [comment]
@cigar-galaxy82 kindly merge this

**Pull request merged**